.github/workflows/labels.yml

```diff
@@ -0,0 +1,14 @@
+name: "Label PR"
+
+on:
+  pull_request_target:
```

Comment:
should we use just `pull_request`? https://securitylab.github.com/research/github-actions-preventing-pwn-requests actions/labels changed to `pull_request_target` to allow forked PRs also receive these labels, which doesn't seem the case for nixpkgs https://github.com/actions/labeler/issues/121#issuecomment-768239617

Reply:
> changed to `pull_request_target` to allow forked PRs also receive these labels

We want PRs from forks to be labeled?

Reply:
oh, you are right! reading it again, I misunderstood their wording. Sorry for the noise.